tabular/src/autogluon/tabular/models/tabular_nn/torch/tabular_nn_torch.py

```diff
@@ -590,16 +592,11 @@ def load(cls, path: str, reset_paths=True, verbose=True):
         model: TabularNeuralNetTorchModel = super().load(path=path, reset_paths=reset_paths, verbose=verbose)
         if model._architecture_desc is not None:
             import torch
-            from .torch_network_modules import EmbedNet
-
-            # recreate network from architecture description
-            model.model = EmbedNet(problem_type=model.problem_type,
-                                   num_net_outputs=model._get_num_net_outputs(),
-                                   quantile_levels=model.quantile_levels,
-                                   architecture_desc=model._architecture_desc,
-                                   device=model.device)
             model._architecture_desc = None
-            model.model = torch.load(model.path + model.params_file_name)
```

Comment:
Naive question: Since we aren't doing anything with architecture_desc anymore here, do we still need to store it?

Reply:
Nice catch!

I removed the architecture_desc, and everything works just fine.